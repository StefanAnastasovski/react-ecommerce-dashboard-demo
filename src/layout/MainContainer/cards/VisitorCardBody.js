import React from "react";

import { FONT_WEIGHT_500 } from "../../../data/constants";
import { DASHBOARD_API } from "../../../data/dashboardConfig";

import Paragraph from "../../../components/Typography/Paragraph";

const VisitorCardBody = () => {
  const styledText = {
    sx: { fontSize: "42px", fontWeight: FONT_WEIGHT_500 },
  };
  const {
    visitors: { data },
  } = DASHBOARD_API;
  return (
    <>
      <Paragraph {...styledText}>{data}</Paragraph>
    </>
  );
};

export default VisitorCardBody;
