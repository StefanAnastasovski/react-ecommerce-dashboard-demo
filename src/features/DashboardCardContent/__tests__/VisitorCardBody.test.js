import React from "react";
import { shallow } from "enzyme";

import VisitorCardBody from "../components/cards/VisitorCardBody";

describe("<VisitorCardBody />", () => {
  describe("render()", () => {
    it("Shallow rendering - MatchSnapshot", () => {
      const component = shallow(<VisitorCardBody />);
      expect(component).toMatchSnapshot();
    });
  });
});
