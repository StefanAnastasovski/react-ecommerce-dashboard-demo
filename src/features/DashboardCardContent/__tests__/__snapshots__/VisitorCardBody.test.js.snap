// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VisitorCardBody /> render() Shallow rendering - MatchSnapshot 1`] = `ShallowWrapper {}`;
